src/renderer/dx/DxRenderer.cpp

```diff
@@ -705,8 +714,9 @@ Microsoft::WRL::ComPtr<IDXGISwapChain1> DxEngine::GetSwapChain()
 {
     RETURN_HR_IF_NULL(E_INVALIDARG, pcoordCursor);
 
-    const SMALL_RECT sr = Microsoft::Console::Types::Viewport::FromCoord(*pcoordCursor).ToInclusive();
-    return Invalidate(&sr);
+    _invalidMap.set(_InvalidToFullRow(til::rectangle{ *pcoordCursor, til::size{1, 1} }));

```

Comment:
does it make more sense to make `_InvalidToFullRow` just `_InvalidateRect` and move the `.set` into it with the row expansion logic?

Reply:
feel free to NAK me

Reply:
Hmmmmmm. The reason I didn't is because I had some dream that the cursor would go back to invalidating an individual cell in the future. But I guess it can just be rearranged again in the future.

Reply:
I think that makes a _better_ case for concentrating the logic, if i'm honest? Having all invalidation setters go through a single function that 1. expands or 2. doesn't expand on set makes it easier to turf the expand logic in one place

Reply:
Sure. I mean, I did it. So I resolved this.